misaligned.js

```diff
@@ -1,16 +1,33 @@
 const {expect} = require('chai')
 
-function print_color_map() {
+function generateColorMap() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
     const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    const colorMap = [];
+
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            colorMap.push({ index: i * 5 + j, major: majorColors[i], minor: minorColors[j] });
         }
     }
-    return majorColors.length * minorColors.length;
+    return colorMap;
+}
+
+function formatEntry(entry) {
+    return `${entry.index} | ${entry.major} | ${entry.minor}`;
 }
 
-result = print_color_map();
-expect(result).equals(25);
-console.log('All is well (maybe!)');
+function printColorMap() {
+    const colorMap = generateColorMap();
+    colorMap.forEach(entry => console.log(formatEntry(entry)));
+    return colorMap.length;
+}
+
+// Strengthen the tests
+const colorMap = generateColorMap();
+expect(colorMap.length).to.equal(25);
+expect(formatEntry(colorMap[0])).to.equal('0 | White | Blue');

```

Comment:
The expression inside the `equal(...)` must be as per the consumer's expectations, not match whatever the code outputs.
As per the color pair documentation, the pair numbers must start with `1`, not `0`.
Pls update to expect the correct output.